modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/servlet/taglib/OpenGraphTopHeadDynamicInclude.java

```diff
@@ -247,7 +252,8 @@ public void include(
 			LayoutSEOLink layoutSEOLink =
 				_layoutSEOLinkManager.getCanonicalLayoutSEOLink(
 					layout, themeDisplay.getLocale(), canonicalURL,
-					alternateURLs);
+					_portal.getAlternateURLs(
```

Comment:
Should we also use `_portal` in [OpenGraphTopHeadDynamicInclude.java#L135](https://github.com/liferay/liferay-portal/blob/b73cf930ca4b920f796f2a3c79d00713b4d70c63/modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/servlet/taglib/OpenGraphTopHeadDynamicInclude.java#L135) to get all the alternate URLs?

Reply:
Nop, this is just the case we want to omit.

we do not want all the alternate urls be added, 
and if we use the _getAlternateURLs_ method it will use the available urls from the server, that omit the __getAvailableLocales(layout)_ that is the point of the story.